<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client_Window</class>
 <widget class="QMainWindow" name="Client_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,4,4,0">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>客户端</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1,0">
       <item>
        <widget class="QLineEdit" name="lineEdit_ca">
         <property name="placeholderText">
          <string>请输入服务器地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cp">
         <property name="placeholderText">
          <string>请输入端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectToSever">
         <property name="text">
          <string>连接服务端</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1,3,1">
       <item>
        <widget class="QTextEdit" name="textEdit_cs">
         <property name="placeholderText">
          <string>接受数据如下</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="c_send">
         <property name="text">
          <string>发送目标数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_ca">
         <property name="placeholderText">
          <string>请输入待发送数据</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="udp_client">
           <property name="text">
            <string>udp连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="c_senddata">
           <property name="text">
            <string>模拟生成温度风速并发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="animation">
           <property name="text">
            <string>动画显示</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="c_stopsend">
           <property name="text">
            <string>停止发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>输出：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_console"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>温度：风速：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
